src/main/kotlin/calculator/Operator.kt

```diff
@@ -0,0 +1,13 @@
+package calculator
+
+enum class Operator(val value: String) {
+
+    PLUS("+"),
+    MINUS("-"),
+    MULTIPLY("*"),
+    DIVIDE("/");
+
+    companion object {
+        fun of(op: String) = values().find { it.value == op }
```

Comment:
```suggestion
        fun from(op: String) = values().find { it.value == op }
```
정적 팩토리 메소드의 네이밍 방식은 관습적으로
- from : 한개의 파라미터
- of : 여러개의 파라미터

입니다.